extension/src/openvic-extension/singletons/SoundSingleton.cpp

```diff
@@ -180,11 +180,12 @@ Ref<AudioStreamWAV> SoundSingleton::get_sound(String const& path) {
 
 // Get a sound by its define name
 Ref<AudioStreamWAV> SoundSingleton::get_sound_stream(String const& path) {
-	if (sfx_define[path].audioStream.has_value()) {
-		return sfx_define[path].audioStream.value();
-	}
+	sfx_define_map_t::iterator it = sfx_define.find(path);
```

Comment:
```suggestion
	const sfx_define_map_t::const_iterator it = sfx_define.find(path);
```
Could this be const? (same for any other similar cases across the repo)

Reply:
It could be but it wouldn't do anything to do so, its just excessively verbose.